test/system/admin/admin_orderable_test.rb

```diff
@@ -52,6 +52,15 @@ def test_invoices_are_orderable
     assert_appears_before('Orphan Profile', 'ACME Inc.')
   end
 
+  def test_order_links_are_with_single_parameter
+    visit admin_auctions_path
+
+    sorting_href = '/admin/auctions?order%5Badmin_auction_decorators.highest_offer_cents%5D=desc'
```

Comment:
- Why hardcoded URL is needed here? Can we use named routes?
- What does this test verify? It is not clear why that `sorting_href` is expected to be present on the page.

Reply:
This test if verify what there is only one sorting parameter in the button URL after we set sorting. Previous behavior was adding sorting params per each button, now it's only one param per button and that's all.